package.json

```diff
@@ -48,10 +48,10 @@
     "govuk-frontend": "^4.6.0",
     "humanize-string": "^3.0.0",
     "js-cookie": "^3.0.5",
-    "maplibre-gl": "^3.3.1",
+    "maplibre-gl": "^3.5.1",
     "read-excel-file": "^5.6.1",
     "route-snapper": "^0.2.1",
     "svelte": "^4.0.0",
-    "svelte-maplibre": "^0.5.0"
+    "svelte-maplibre": "github:dabreegster/svelte-maplibre#changes_for_atip"
```

Comment:
Switching back to my fork because https://github.com/dimfeld/svelte-maplibre/pull/92 won't be reviewed for about a week while maintainer is on holiday.

Also note this updates svelte-maplibre and fixes some HMR issues! #359